examples/subplots_axes_and_figures/axes_margins.py

```diff
@@ -1,6 +1,6 @@
 """
 =====================================================================
-Zooming in and out using Axes.margins and the subject of "stickiness"
+Axes.margins and "stickiness"
```

Comment:
I'd go for "Controlling view limits using margins and sticky_edges".

That's more informative and still short enough.

Reply:
Will do!